mod/notifications/start.php

```diff
@@ -41,6 +41,10 @@ function notifications_plugin_init() {
 function notifications_page_handler($page) {
 
 	elgg_gatekeeper();
+
+	// Set the context to settings
+	elgg_set_context('settings');
+
```

Comment:
IMO, we need to move all the junk out of pagesetup and not worry about the global scope when resources are rendered.